<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ins_devicechoose</class>
 <widget class="QDialog" name="ins_devicechoose">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #303539</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>请选择一个可用的光源设备：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listW_Devices">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true"> background-color: #333333; </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="title">
      <string>光源设备信息</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="3,8">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>设备码:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="deviceDescription">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="3,8">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>MAC地址:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="deviceMac">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="3,8">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>IP地址:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="deviceIP">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="3,8">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>子网掩码:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="deviceSubnet">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>15</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="PB_Search">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">#PB_Search
{
border-radius:8px;
color: #000000;
background-color: #ffffff;
 

}

#PB_Search::hover
{	
color: #FFFFFF;

background-color: #1781b5;

}

#PB_Search::pressed,QPushButton::checked
{
    color: #FFFFFF;
   background-color: #718093;
}</string>
       </property>
       <property name="text">
        <string>搜索设备</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PB_Select">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">#PB_Select
{
border-radius:8px;
color: #000000;
 background-color: #ffffff; 

}

#PB_Select::hover
{	
color: #FFFFFF;

background-color: #1781b5;

}

#PB_Select::pressed,QPushButton::checked
{
    color: #FFFFFF;
   background-color: #718093;
}</string>
       </property>
       <property name="text">
        <string>选择设备</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PB_Close">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">#PB_Close
{
border-radius:8px;
color: #000000;
 background-color: #ffffff; 

}

#PB_Close::hover
{	
color: #FFFFFF;

background-color: #1781b5;

}

#PB_Close::pressed,QPushButton::checked
{
    color: #FFFFFF;
   background-color: #718093;
}</string>
       </property>
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="handipBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">#handipBtn
{
border-radius:8px;
color: #000000;
 background-color: #ffffff; 

}

#handipBtn::hover
{	
color: #FFFFFF;

background-color: #1781b5;

}

#handipBtn::pressed,QPushButton::checked
{
    color: #FFFFFF;
   background-color: #718093;
}</string>
       </property>
       <property name="text">
        <string>更新 IP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LineUpdateIp">
       <property name="styleSheet">
        <string notr="true">color:#ffffff;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
